explainer-server/app/views/templates/main.scala.html

```diff
@@ -42,6 +42,21 @@
           }
         })
     </script>
+    <script type="text/javascript">
+        /*
+         * Note: there doesn't seem to be a way to acheive the update with an event (for instance onkeypress) on
+         * #explainer-editor__body-wrapper__input
+         * This explains the choice of setInterval
+         */
+        function updateWordCountDisplay(){

```

Comment:
This looks like a plain javascript function now. Could you write it in scala js?